alembic/runtime/environment.py

```diff
@@ -401,7 +402,15 @@ def get_x_argument(
         else:
             value = []
         if as_dictionary:
-            value = dict(arg.split("=", 1) for arg in value)
+            dict_value = {}
+            for arg in value:
+                try:
+                    x_key, x_value = arg.split("=", 1)
```

Comment:
we could more simply use partition here, without the try catch
`x_key, _, x_value = arg.partition('=')`

Reply:
Thanks! I always forget about `partition`. Fixed!

Reply:
me too, remembered after a while